### PR TITLE
Show message while loading itempanel

### PR DESCRIPTION
**Pull request by miozune**

### CODE REVIEW

**Comment by slprime** [comment]
It should be in `ItemsGrid`

**Comment by slprime** [comment]
I'm not sure about the message in bookmarks. it will be annoying

**Comment by miozune** [comment]
hmm okay

File: src/main/resources/assets/nei/lang/en_US.lang [hunk 1]
```diff
@@ -11,6 +11,7 @@ nei.bookmark.toggle.tip=Toggle visibility of the Bookmark Panel
 nei.bookmark.viewmode.toggle.tip=Toggle Bookmark View Mode
 
 nei.itempanel.quantity.default=One stack
+nei.itempanel.loading=Loading ItemList...
```
Comment (embeddedt) [inline src/main/resources/assets/nei/lang/en_US.lang:14]: This looks better to a non-technical player IMO.

```suggestion
nei.itempanel.loading=Loading Item List...
```

**Comment by slprime** [comment]
what about Label.java?

**Review by slprime — APPROVED** [review]

**Pull request merged**